CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# These owners will be the default owners for everything in the repo.
+# Unless a later match takes precedence, they will be requested for review when someone
+# opens a pull request.
+*       @blootsvoets @nivemaham @fnobilia
```

Comment:
I would prefer tags being used for this. Git-flow already proposes tagging releases. A tag is just a named commit, just like the head of a branch is a named commit so essentially the two are equivalent.

But a branch has more the meaning of an active piece of work, while a tag has the meaning of an important point in history. You can always check out a tag, just like you can check out a branch, if you need to do fixes to previous releases.